Privacy.md

```diff
@@ -0,0 +1,56 @@
+# BlitzMail Privacy Policy
+This is the developer's policy which is created to give details and informs you if your personal information is collected, used, disclosed and secured by the developer. These policies are stated to ensure you know what personal details you may or may not disclose when the software is being utilized and the choices you have associated with that data. If you decide to continue using this software, it means you accept all terms and conditions stated by the developer in this policy. <hr>
```

Comment:
This could be shortened a lot, especially the 'you may or may not disclose' part.